src/common/entity/binary_sensor_icon.ts

```diff
@@ -16,9 +16,13 @@ export const binarySensorIcon = (state: HassEntity) => {
     case "garage_door":
       return activated ? "hass:garage" : "hass:garage-open";
     case "gas":
+      return activated ? "hass:shield-check" : "hass:biohazard";
```

Comment:
gas really isn't a biohazard, is it? We could just leave it how it was.

Reply:
thought I had done that, but @kukulich said I changed the icon...
what about mdi:smoke-detector?
```
return activated ? "hass:shield-check" : "hass:smoke-detector";
```

Reply:
If we don't have something specific lets just fall back to the default like smoke?

Reply:
right, entering:
```
      return activated ? "hass:shield-check" : "hass:alert";
```